ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -675,6 +682,8 @@ ethernet_interfaces:
     speed: < interface_speed >
     mtu: < mtu >
     type: < routed | switched >
+        logging_event:
```

Comment:
indention looks wrong here

Reply:
Thanks for the spot. 
It requires to update this element to fit with PR#570 as well (1bb9df3)